sample/halles_des_vosges/README.md

```diff
@@ -0,0 +1,23 @@
+# Halles des Vosges, Belfort, France
+
+This sample is a demonstration of multi-material building and slopped roof.
```

Comment:
"… is a demonstration of how to model a multi-material building with slopped roof"